src/main/java/net/masterthought/cucumber/generators/FailuresOverviewPage.java

```diff
@@ -16,6 +21,19 @@ public String getWebPage() {
 
     @Override
     public void prepareReport() {
-        context.put("all_features", report.getAllFeatures());
+        List<Element> failures = new ArrayList<>();
+        for (Feature feature : report.getAllFeatures()) {
+            if (feature.getStatus().isPassed()) continue;
+
+            for (Element element : feature.getElements()) {
+                if (!element.getBeforeStatus().isPassed()
+                    || !element.getStepsStatus().isPassed()
+                    || !element.getAfterStatus().isPassed()) {
```

Comment:
Hm. It depends. Before steps are usually to setup a test environment. It might be important to know that those failed. In the same manner, after steps are generally used to cleanup the test environment/data. It might also be good to know that failed as well.